File: conditionals/old_enough.py
# Elizabeth Gutierrez, Old Enough in Python
age = int(input("How old are you?:\n"))
if age >= 18:
    print("You can vote!")
elif age >= 16:
    print("You can drive!")
elif age >= 15:
    print("You can get your permit!")
elif age >= 6:
    print("You can go to school!")
else:
    print("You cannot go to school")

File: conditionals/old_enough.c
// Elizabeth Gutierrez, Old Enough in C
#include <stdio.h>
#include <string.h>

int main() {
    int age;
    printf("How old are you?:\n");
    scanf("%d", &age);

    if (age >= 18) {
        printf("You can vote!\n");
    } else if (age >= 16) {
        printf("You can drive!\n");
    } else if (age >= 15) {
        printf("You can get your permit!\n");
    } else if (age >= 6) {
        printf("You can go to school!\n");
    } else {
        printf("You cannot go to school\n");
    }

    return 0;
}

